package.json

```diff
@@ -5,7 +5,9 @@
     "./plugin",
     "./test-site"
   ],
-  "devDependencies": {},
+  "devDependencies": {
+    "@types/request": "^2.48.5"
+  },
```

Comment:
Was this added by mistake?

Reply:
I suppose so - not really sure when this was added; certainly didn't intend for it. My bad! I'll clean it up :+1: